catalog/v4.15/Containerfile

```diff
@@ -1,6 +1,6 @@
 # The base image is expected to contain
 # /bin/opm (with a serve subcommand) and /bin/grpc_health_probe
-FROM registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.15
+FROM registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.18
```

Comment:
This seems to be working in the builds, but I'm also not sure why we'd use the 4.18 container image to build the index for 4.15? I was under the impression the catalog and index should align with the builder:

catalog/v4.15/Containerfile use registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.15